contracts/ColonyTask.sol

```diff
@@ -266,13 +266,12 @@ contract ColonyTask is ColonyStorage, DSMath {
     return taskWorkRatings[_id].secret[_role];
   }
 
-  // TODO: Restrict function visibility to whoever submits the approved Transaction from Client
   // Note task assignment is agreed off-chain
   function setTaskRoleUser(uint256 _id, uint8 _role, address _user) public
+  confirmTaskRoleIdentity(_id, MANAGER)
```

Comment:
Can we refactor `isManager` to `confirmTaskRoleIdentity` as it is doing the same thing?